packages/mdc-radio/component.ts

```diff
@@ -64,6 +59,20 @@ export class MDCRadio extends MDCComponent<MDCRadioFoundation> implements MDCRip
     return this.ripple_;
   }
 
+  private get nativeControl_(): HTMLInputElement {
+    const {NATIVE_CONTROL_SELECTOR} = MDCRadioFoundation.strings;
+    const el = this.root_.querySelector<HTMLInputElement>(NATIVE_CONTROL_SELECTOR);
+    if (!el) {
+      throw new Error(`Radio component requires a ${NATIVE_CONTROL_SELECTOR} element`);
+    }
+    return el;
+  }
+
+  // Public visibility for this property is required by MDCRippleCapableSurface.
```

Comment:
Why is this down below something that's private now?

Reply:
I was trying to make radio and checkbox consistent.
Moved `private get nativeControl_()` to the bottom of both files.